src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -269,7 +274,7 @@ def _decide_cancel_submission(upload_group: AssetUploadGroup) -> bool:
     "parsed/consumed by custom scripts.",
 )
 @_handle_error
-def bundle_gui_submit(job_bundle_dir, browse, output, install_gui, **args):
+def bundle_gui_submit(job_bundle_dir, browse, output, install_gui, submitter_name, **args):
```

Comment:
Does the non-gui version also need this?

Reply:
The regular bundle submit command doesn't have a GUI and doesn't need the dependencies right?

Reply:
Correct, we do not bring in GUI deps for the CLI.

Reply:
Sorry I got mixed up with another PR. Makes sense, will update.